### ISSUE TITLE
The installation guide should be easier to get to

### ISSUE THREAD

**Issue by jwinnie** [elementary/website#1730]
Currently, a link to the [installation instructions](https://elementary.io/docs/installation) is shown on the popup before the user downloads elementaryOS. However, users might be inclined 
to just click the big, colorful button without reading the text above it. Furthermore, users might want to download first, while they have the dialog open, and follow the instructions later. After users click the button, they are abruptly brought back to the home page. Then, they might be confused because they can't open the disk image with their file manager. Worse, they might find unsafe or difficult instructions online. In any case, it should be more obvious that there is an easy installation guide available.

Possible solutions:
- Make the "installation guide" link more obvious (make it bigger, more colorful, etc)
- Automatically redirect the user to the installation guide after the "download" button is clicked (the user probably doesn't need the homepage after installing anyway)
- Show a page after downloading that says something like "Thank you for downloading! To install, follow [these instructions]()"

**Comment by josegatica** [comment]
I can take this issue.

**Comment by brunocamarggo** [comment]
I tried to fix this issue but I don't know how to redirect when the user chooses the magnet link option. 

I tried using...
`window.location.replace('thank-you-page')`, `window.location.href = "/thank-you-page"` and `window.location = "thank-you-page"`

I mean, when the popup "Launch Application" is shown, the browser simply does not redirect to "Thank-you-page". The same  javascript function is used (and works) to the others ways to download.

Someone knows why this happens? Thanks a lot.
